### PR TITLE
Added missing fields to publication list commands

### PR DESCRIPTION
**Pull request by gerrod3**
fixes: #207

### CODE REVIEW

File: pulpcore/cli/common/generic.py [hunk 1]
```diff
@@ -271,6 +271,42 @@ def pulp_option(*args: Any, **kwargs: Any) -> Callable[[_F], _F]:
...
+pulp_created_lte_option = pulp_option(
+    "--created-lte",
+    "pulp_created__lte",
+    help=_("Search for {entities} created at or before this ISO 8601 date"),
+    type=str,
+)
+
+publication_filter_options = [
+    content_in_option,
+    pulp_created_exact_option,
```
Comment (mdellweg) [inline pulpcore/cli/common/generic.py:304]: An exact date to the millisecond does not sound like a useful filter to me.

File: pulpcore/cli/file/publication.py [hunk 2]
```diff
@@ -49,8 +50,12 @@ def publication(ctx: click.Context, pulp_ctx: PulpContext, publication_type: str
     click.option(
         "--version", type=int, help=_("a repository version number, leave blank for latest")
     ),
+    click.option(
+        "--manifest",
+        help=_("Filename to use for manifest file containing metadata for all the files."),
+    ),
```
Comment (mdellweg) [inline pulpcore/cli/file/publication.py:56]: Not technically the same, but OK...

File: pulpcore/cli/common/generic.py [hunk 3]
```diff
@@ -271,6 +271,42 @@ def pulp_option(*args: Any, **kwargs: Any) -> Callable[[_F], _F]:
...
 
+content_in_option = pulp_option(
+    "--content",
+    "content__in",
+    help=_("Search for {entities} with these content hrefs in them (JSON list)"),
+    callback=load_json_callback,
+)
+
+pulp_created_exact_option = pulp_option(
+    "--created-exact",
```
Comment (mdellweg) [inline pulpcore/cli/common/generic.py:282]: What about "created at"? And do you think we could add some fuzzyness? If i specify the date to the day, i might want to see publications made that day.

On a related note, the others could be named "creted before" and "created after".

File: pulpcore/cli/file/publication.py (hunk 2 above)
Comment (daviddavis) [reply pulpcore/cli/file/publication.py:56]: I believe we should specify pulp_file >= 1.7.0 for this option. 

Also I don’t see this change reflected in the change log.

Comment (daviddavis) [reply pulpcore/cli/file/publication.py:56]: Oh sorry I thought this was the repository command. Ignore my comment about 1.7.0.

**Review by mdellweg — APPROVED** [review]

**Comment by mdellweg** [comment]
The CI failure looks like #259 to me. Rebase should fix it.

**Comment by mdellweg** [comment]
What's going on here?
https://github.com/pulp/pulp-cli/pull/253/checks?check_run_id=2765262634#step:5:1097

**Pull request merged**